Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -1146,7 +1146,7 @@ def commit(self):
                 selected_data = data[mask]
                 if self.append_clusters:
                     def remove_other_value(vars_):
-                        vars_ = [var for var in vars_]
+                        vars_ = list(vars_).copy()
```

Comment:
Ooops, in my earlier comment I thought that `vars_` was a list. Sorry. Now that you have to convert it to a list anyway, `list(vars_)` will suffice; you can skip `copy()`.